### ISSUE TITLE
Webpage should fit both desktop and android device screen

### ISSUE THREAD

**Issue by gouri-panda** [gouri-panda/BudgetApp#3]
* Describe the bug
Right now the website is only compatible with the desktop device
* Expected behavior
The website should both compatible with android and desktop

**Comment by BhuvaneshPatil** [comment]
I have tried making it responsive..
![Budgety - Google Chrome 04-10-2020 21_06_37](https://user-images.githubusercontent.com/27822551/95019912-ba3e6380-0685-11eb-8240-92c037470684.png)

**Comment by BhuvaneshPatil** [comment]
shall I make pull request?

**Comment by gouri-panda** [comment]
yes @BhuvaneshPatil if you have completed your task.

**Comment by deepharshsri** [comment]
Can we use here materialize or bootstrap for styling?

**Comment by gouri-panda** [comment]
yes @deepharshsri sure open an issue and then start solving the issue and create PR.

**Comment by chandresh1339** [comment]
@gouri-panda,Would you like to add the dark/light theme to BudgetApp to make it more interactive. I can hopefully submit my first PR.
